cfg/rh-1.0/master.yaml

```diff
@@ -129,7 +129,7 @@ groups:
         scored: false
 
       - id: 1.1.8
-        text: "Ensure that the etcd pod specification file ownership is set to root:root (Automated)"
+        text: "Ensure that the etcd pod specification file ownership is set to root:root (Manual)"
```

Comment:
Should update the value of `scored` field to `false` if it should be `Manual`.

Reply:
I mean this field:
<img width="911" alt="image" src="https://user-images.githubusercontent.com/485054/189661641-5551d403-1605-4873-bb9c-2e5fc8762e13.png">